lib/puma/const.rb

```diff
@@ -100,7 +100,7 @@ class UnsupportedOption < RuntimeError
   # too taxing on performance.
   module Const
 
-    PUMA_VERSION = VERSION = "4.3.0".freeze
+    PUMA_VERSION = VERSION = "4.3.1".freeze
```

Comment:
I think you may need to rebase with master to get rid of this change. In general we'll rev the version.